axelrod/strategies/cycler.py

```diff
@@ -1,13 +1,15 @@
-from axelrod.actions import Actions, Action
+from axelrod.actions import Actions, Action, str_to_actions
 from axelrod.player import Player
-
+import itertools
 import copy
 
+C, D = Actions.C, Actions.D
+
 
 class AntiCycler(Player):
     """
     A player that follows a sequence of plays that contains no cycles:
-    C CD CCD CCCD CCCCD CCCCCD ...
+    CDD  CD  CCD CCCD CCCCD ...
```

Comment:
the first six to eight moves are a cycle.  CCDCCD(CC).  

are you saying it should have beem D CD CCD CCCD CCCCD?  the original was what the old docstring said. 

D CD CCD ... would still trigger detect_cycle on round 4.